src/serlo-editor-repo/plugin-text/components/text-editor.tsx

```diff
@@ -176,10 +176,7 @@ export function TextEditor(props: TextEditorProps) {
 
         if (node && parent) {
           if (Object.hasOwn(parent, 'type') && parent.type === 'a') {
-            if (
-              Object.hasOwn(node, 'text') &&
-              node.text.length - 1 === offset
-            ) {
+            if (Object.hasOwn(node, 'text') && node.text.length - 1 <= offset) {
```

Comment:
@hejtful I ran into an edge case here. I think when the link is the last item in a line i could not move out of the link with `ArrowRight`. This way works for me. Does my change makes sense to you?